src/posix/platform/udp.cpp

```diff
@@ -79,15 +79,19 @@ static bool IsMulticast(const otIp6Address &aAddress)
 
 static otError transmitPacket(int aFd, uint8_t *aPayload, uint16_t aLength, const otMessageInfo &aMessageInfo)
 {
+#ifdef __APPLE__
+    constexpr size_t kBufferSize = 128;
```

Comment:
Is it possible to use some `sizeof(xxx)` ?

Reply:
I don't know what xxx is.

Reply:
I think Simon is doesn't like that, on mac OS, we're using a magic number (128) where on other platforms, you're using some macros that explain how big the buffer is (CMSG_SPACE of particular structures).  If I understand, the reason you're doing it this way is because mac OS generates a warning/error about the expression supplied?  Maybe a comment that says why we have to do it this way on mac OS?  Or did I not understand the change?

Reply:
Agree that at least a comment is needed.